full/Dockerfile

```diff
@@ -253,6 +253,17 @@ RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
 
 RUN bash -lc "cargo install cargo-watch"
 
+### VLANG ###
+USER root
+ENV VLANG_SOURCE="/opt/vlang"
+RUN git clone https://github.com/vlang/v "$VLANG_SOURCE" || exit 1
```

Comment:
Maybe you should use a stable release instead of nightly?

Reply:
rafactored, better version selection adapted